_functions.sh

```diff
@@ -2246,7 +2269,10 @@ do_undeploy() {
       # Close firewall port for LDAPS
       do_ufw_close_port ${DEPLOYMENT_LDAP_PORT} "Ldap Port" ${ADT_DEV_MODE}
     fi
-    
+    if [ "${DEPLOYMENT_IFRAMELY_ENABLED}" == "true" ]; then
+      # Close firewall port for IFRAMELY
+      do_ufw_close_port ${DEPLOYMENT_IFRAMELY_PORT} "IFRAMELY Port" ${ADT_DEV_MODE}
```

Comment:
Why do we need this?

Reply:
Not really, it was taken from the ldap conf